types.ts

```diff
@@ -416,6 +665,10 @@ export type BasicPackDefinition = Omit<PackVersionDefinition, 'version'>;
  * heart of the implementation of a Pack.
  */
 export interface PackVersionDefinition {
+  /**
+   * The semantic version of the pack. This must be valid semantic version of the form `1`, `1.2`, or `1.2.3`.
+   * When uploading a pack version, the semantic version must be greater than any previously uploaded version.
+   */
```

Comment:
Should we mention somewhere, perhaps in the interface-level description, the need / behavior of these versions in the Web IDE.

Reply:
Up to you. The web IDE uses BasicPackVersionDefinition which doesn't have a version at all, so my thinking was that a web IDE user wouldn't come across this interface on their own.

Reply:
Ah, I didn't realize there was a separate type for the Web IDE.